src/BenchmarkDotNet/Portability/RuntimeInformation.cs

```diff
@@ -42,6 +42,13 @@ public static bool IsCoreRT
             => ((Environment.Version.Major >= 5) || FrameworkDescription.StartsWith(".NET Core", StringComparison.OrdinalIgnoreCase))
                && string.IsNullOrEmpty(typeof(object).Assembly.Location); // but it's merged to a single .exe and .Location returns null here ;)
 
+        /// <summary>
+        /// "Is the target where we will run the benchmarks WASM?"
+        /// </summary>
+        public static bool IsBenchmarkTargetWasm { get; set; }
```

Comment:
benchmark host can run benchmarks against multiple runtimes by using things like:

```cmd
--runtimes net48 netcoreapp3.1 wasm
```

so this static field would introduce a bug. Please remove it
```suggestion
```